ch09-writing-your-own-methods/old_school_roman_numerals.rb

```diff
@@ -1,3 +1,34 @@
-def old_roman_numeral num
-  # your code here
-end
\ No newline at end of file
+
+class ToRomans
+
+  VALUES = { "I" => 1, 
+             "V" => 5,
+             "X" => 10,
+             "L" => 50,
+             "C" => 100,
+             "D" => 500,
+             "M" => 1000
+  }
+
+  def old_roman_numeral number
+    calculator number
+  end
+
+private
```

Comment:
I recommend indenting as per the style guide: https://github.com/bbatsov/ruby-style-guide#indent-public-private-protected